tests/backup/backup_license_test.go

```diff
@@ -415,12 +412,10 @@ var _ = Describe("{LicensingCountBeforeAndAfterBackupPodRestart}", func() {
 			err = DeletePodWithLabelInNamespace(pxbNamespace, nil)
 			dash.VerifyFatal(err, nil, "Restart all the backup pods")
 			log.InfoD("Validate if all the backup pods are up")
-			pods, err := core.Instance().GetPods(pxbNamespace, nil)
-			dash.VerifyFatal(err, nil, "Getting all the backup pods")
-			for _, pod := range pods.Items {
-				err = core.Instance().ValidatePod(&pod, podReadyTimeout, podReadyRetryTime)
-				log.FailOnError(err, fmt.Sprintf("Failed to validate pod [%s]", pod.GetName()))
-			}
+			err = ValidateAllPodsInPxBackupNamespace()
+			log.FailOnError(err, "Failed to validate pods in px-backup namespace")
+			err = IsMongoDBReady()
```

Comment:
Let us use this inside `ValidateAllPodsInPxBackupNamespace` instead of calling it in every test case.

Reply:
done